src/middlewared/middlewared/alert/source/ctdb.py

```diff
@@ -1,4 +1,9 @@
-from middlewared.alert.base import AlertCategory, AlertClass, AlertLevel, SimpleOneShotAlertClass
+from datetime import timedelta
+from middlewared.alert.base import Alert, AlertCategory, AlertClass, AlertLevel, AlertSource, SimpleOneShotAlertClass
+from middlewared.alert.schedule import IntervalSchedule
+
+ALLOWED_OFFSET_CLOCK_REALTIME = 120
```

Comment:
Just curious but do you think we should store these 2 global options somewhere in the cluster plugin? Idk, can't really decide. I'll leave it up to you 😄